include/nrf9160_time.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef NRF9160_TIME_H__
+#define NRF9160_TIME_H__
+
+#include <zephyr/types.h>
+#include <time.h>
+
+/**
+ * @defgroup nrf9160_time nRF9160 Time Library
+ * @{
+ * @brief Library that maintains the current date time UTC that can be fetched
+ *        in order to timestamp data.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Initialize the nRF9160 time library.
+ */
+void nrf9160_time_init(void);
+
+/** @brief Set the current date time UTC.
+ *
+ *  @param[in] new_date_time Pointer to a tm structure.
+ */
+void date_time_set(const struct tm *new_date_time);
```

Comment:
IMO it would be better to use some common prefix for all these functions.

Reply:
I agree. Will change when the name for the library has been agreed upon.